### DISCUSSION TITLE
How to specify library exports?

### DISCUSSION THREAD

**Discussion by rzvc** [vimeo/psalm#7954] [Q&A] [answered]
When analyzing a library, how can you specify library exports, so `PossiblyUnusedMethod` doesn't get triggered for everything that is meant for outside consumption?

**Comment by AndrolGenhald** [comment, marked as answer]
Unfortunately this isn't well supported at the moment. The inverse can be annotated with `@internal`, but for now I would recommend adding an [issue handler](https://psalm.dev/docs/running_psalm/dealing_with_code_issues/#config-suppression) for it. See also #7456.

**Reply by rzvc** [reply]
Thanks, I will give it a try, and subscribed to that issue in case support for libraries is added.
